src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -618,11 +618,20 @@ uct_rc_mlx5_ep_connect_qp(uct_rc_mlx5_iface_common_t *iface,
                           uct_ib_mlx5_qp_t *qp, uint32_t qp_num,
                           struct ibv_ah_attr *ah_attr)
 {
+    uct_ib_mlx5_qp_connect_attr_t con_attr;
+
     switch (qp->type) {
     case UCT_IB_MLX5_QP_TYPE_VERBS:
         return uct_rc_iface_qp_connect(&iface->super, qp->verbs.qp, qp_num, ah_attr);
     case UCT_IB_MLX5_QP_TYPE_DEVX:
-        return uct_ib_mlx5_devx_connect_qp(&iface->super.super, qp, qp_num, ah_attr);
+        con_attr.timeout       = iface->super.config.timeout;
+        con_attr.rnr_retry     = iface->super.config.rnr_retry;
+        con_attr.retry_cnt     = iface->super.config.retry_cnt;
+        con_attr.max_rd_atomic = iface->super.config.max_rd_atomic;
+        con_attr.min_rnr_nak   = iface->super.config.min_rnr_timer;
+        con_attr.exp_backoff   = iface->super.config.exp_backoff;
```

Comment:
maybe pass uct_rc_mlx5_common_t instead of defining new struct?
the other function (uct_rc_iface_qp_connect) also has rc specific arg

Reply:
Making uct/ib/mlx5 be circularly depended on uct/ib/rc? 
We can't include include rc_mlx5_common.h (defining uct_rc_mlx5_iface_common_t) from ib_mlx5.h (defining uct_ib_mlx5_devx_connect_qp). May move uct_ib_mlx5_devx_connect_qp under src/ib/rc somewhere...

Reply:
do we expect to have connect_qp for anything other than rc/dc?
if not lets move it to rc_mlx5_common.h as well